Snippets/scale-upem.py

```diff
@@ -0,0 +1,156 @@
+from fontTools.ttLib.ttVisitor import TTVisitor
```

Comment:
could you add a small module docstring at the top briefly summarizing what this scale-upem script does (and does not) currently support?